MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -2024,11 +2024,11 @@
 	_properties.limits.timestampPeriod = mvkGetTimestampPeriod();
 
     _properties.limits.pointSizeRange[0] = 1;
-    _properties.limits.pointSizeRange[1] = 64;
+    _properties.limits.pointSizeRange[1] = 511;
```

Comment:
I lowered this because the actual maximum isn't 511 as documented, but varies from device to device. Changing this causes point rendering tests to fail later on.

Reply:
Thanks. Reverted.

I guess at some point we could enumerate test major devices and set accordingly.